src/components/pill.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC, ReactNode } from "react"
+import classnames from "classnames"
+
+interface Props {
+  children: ReactNode
+  secondary?: boolean
+}
+
+export const Pill: FC<Props> = ({ children, secondary }) => (
```

Comment:
fyi the correct name is `badge` for this kind of component

Reply:
thanks! motivated by your comment, I aligned with Enrique on the naming. he thinks `pill` is appropriate. bootstrap calls these `pill badges`, so we're both kinda close :) https://getbootstrap.com/docs/4.0/components/badge/#pill-badges

I'd argue that `badge` is reserved for dealer badges, mbg badge, etc already